### PR TITLE
Librewolf: Add version 95.0.2

### LINKED ISSUE #6333: [Request] Add Librewolf
>LibreWolf
>A fork of Firefox, focused on privacy, security and freedom.

https://librewolf-community.gitlab.io/

Windows release repository https://gitlab.com/librewolf-community/browser/windows

### PR DESCRIPTION
**Pull request by ItsPizzaTime1501**
Librewolf is just a fork of Firefox that is a lot more privacy focused. As far as I can tell there's no way to auto-update due to how gitlab handles release urls but if someone knows how to feel free to make changes to this PR. (Also sorry if this is scuffed in any way this is my first package)

Closes #6333

- [X] I have read the [Contributing Guide](https://github.com/ScoopInstaller/.github/blob/main/.github/CONTRIBUTING.md).

### CODE REVIEW

**Comment by rashil2000** [comment]
Use this:

```json
    "checkver": {
        "url": "https://gitlab.com/api/v4/projects/13852981/releases",
        "jsonpath": "$[0].assets.links[1].url",
        "regex": "/uploads/(?<randid>[a-z0-9]+)/librewolf-([\\d.]+)\\.en-US\\.win64\\.zip"
    }
```

Since this is portable, anything to persist? Where are user settings stored?

**Comment by ItsPizzaTime1501** [comment]
Guess I forgot about keeping user settings, everything is stored in the Profiles folder. Will fix that right away and add what you suggested.

**Comment by rashil2000** [comment]
Add autoupdate too

**Comment by ItsPizzaTime1501** [comment]
Could you maybe explain how you'd go about creating an autoupdate for this cause I can't quite figure it out with the random 32 char string in the url. I've been looking at the wiki and messing with stuff for about an hour now.

**Comment by rashil2000** [comment]
In the checkver regex, I captured a variable called `randid` for you. You can use that like `$matchRandid`.

**Comment by ItsPizzaTime1501** [comment]
Latest commit seems to autoupdate fine from my tests, I removed the hash cause the only place that has the hash is the gitlab release which would mean getting another randid. If it seems necessary to get the hash feel free to tell me and I'll see what I can do.

File: bucket/librewolf.json [hunk 1]
```diff
@@ -4,7 +4,6 @@
     "homepage": "https://librewolf.net/",
     "license": "MPL-2.0",
     "url": "https://gitlab.com/librewolf-community/browser/windows/uploads/090a5145a8e3b3234f5c215ae26d61cb/librewolf-95.0.2.en-US.win64.zip",
-    "hash": "63a42ac89266004066efc412111bdc1f29a6de18df75e04faa61ccf45758c141",
```
Comment (rashil2000) [inline bucket/librewolf.json]: Hash is necessary.

File: bucket/librewolf.json [hunk 2]
```diff
@@ -22,6 +21,10 @@
     "checkver": {
         "url": "https://gitlab.com/api/v4/projects/13852981/releases",
         "jsonpath": "$[0].assets.links[1].url",
-        "regex": "/uploads/(?<randid>[a-z0-9]+)/librewolf-([\\d.]+)\\.en-US\\.win64\\.zip"
+        "regex": "/uploads/(?<randid>[a-z0-9]+)/librewolf-(?<version>[\\d.]+)\\.en-US\\.win64\\.zip"
```
Comment (rashil2000) [inline bucket/librewolf.json:24]:
```suggestion
        "regex": "/uploads/(?<randid>[a-z0-9]+)/librewolf-([\\d.]+)\\.en-US\\.win64\\.zip"
```

Version is automatically captured, so no need to name it

File: bucket/librewolf.json [hunk 3]
```diff
@@ -22,6 +21,10 @@
...
+    },
+    "autoupdate": {
+        "url": "https://gitlab.com/librewolf-community/browser/windows/uploads/$matchRandid/librewolf-$matchVersion.en-US.win64.zip",
+        "extract_dir": "librewolf-$matchVersion"
```
Comment (rashil2000) [inline bucket/librewolf.json:28]:
```suggestion
        "url": "https://gitlab.com/librewolf-community/browser/windows/uploads/$matchRandid/librewolf-$version.en-US.win64.zip",
        "extract_dir": "librewolf-$version"
```

For version, we don't need to prefix `match`

**Review by rashil2000 — CHANGES_REQUESTED** [review]

File: bucket/librewolf.json (hunk 2 above)
Comment (ItsPizzaTime1501) [reply bucket/librewolf.json:24]: ah, I saw them capture version in the openjdk example and assumed it was necessary

File: bucket/librewolf.json (hunk 1 above)
Comment (ItsPizzaTime1501) [reply bucket/librewolf.json]: so does that mean I have to add hash to autoupdate?

**Comment by ItsPizzaTime1501** [comment]
Autoupdate seems to re-calculate the hash by itself so I don't have to grab it from the gitlab releases. Hopefully that's fine.

File: bucket/librewolf.json (hunk 1 above)
Comment (rashil2000) [reply bucket/librewolf.json]: It's slightly tricky. Let me take a look.

**Pull request merged**